build.gradle

```diff
@@ -25,6 +25,7 @@ plugins {
 
 ext.ballerinaLangVersion = project.ballerinaLangVersion
 ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
+ext.checkstyleToolVersion = project.checkstyleToolVersion
 ext.slf4jVersion = project.slf4jVersion
 ext.axiomVersion = project.axiomVersion
 ext.jakartaActivationVersion = project.jakartaActivationVersion
```

Comment:
```suggestion
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-xslt/pull/48/commits/c144fbc301c49f7ebd902ad022e9be4c2fb05b27